lib/Pipes/TaggedFunctionKind.cpp

```diff
@@ -31,7 +31,7 @@ using namespace llvm;
 
 std::optional<pipeline::Target>
 TaggedFunctionKind::symbolToTarget(const llvm::Function &Symbol) const {
-  if (not Tag->isTagOf(&Symbol) or Symbol.isDeclaration())
+  if (not Tag->isExactTagOf(&Symbol) or Symbol.isDeclaration())
```

Comment:
Here we're fixing a long standing bug, but the commit does not mention it.

The commit message just says "Preliminaries" and includes a set of changes that do not seem strictly related to me. I think one commit can be the one fixing this bug and the other the `stopTracking` thing.